Sources/Hummingbird/Utils/DateCache+Date.swift

```diff
@@ -17,11 +17,11 @@ import Foundation
 extension HBDateCache {
     /// Date formatter as per section 5.2.14 of RFC 1123
     /// https://www.rfc-editor.org/rfc/rfc1123#page-55
-    static let rfc1123Formatter: DateFormatter = {
+    static var rfc1123Formatter: DateFormatter {
```

Comment:
I assume there were some warnings around this?

You could just use `nonisolated(unsafe)`, considering `DateFormatter`s are known to be expensive, and this is also internal so you know you're not modifying it elsewhere.

Alternatively you could wrap it in another type to avoid mutations, but not sure if that's worth that effort compared to using `nonisolated(unsafe)`.

Reply:
Unfortunately we can't really use a global here. DateFormatter on Linux is not thread safe. See https://github.com/apple/swift-corelibs-foundation/issues/3166

Reply:
I wonder if it is still worth optimizing for.
I don't think what Hummingbird does can trigger that race condition, but even then, a `NIOLockedValueBox` might be better than just making it a computed property.

Reply:
Generally speaking, yes, but based on the code, any locking mechanism should be plenty to avoid any crashes in a DateFormatter 🤔

It is suboptimal though, I admit.